tools/networkmonitor/networkmonitor.nim

```diff
@@ -4,17 +4,18 @@ else:
   {.push raises: [].}
 
 import
-  std/[tables,strutils,times,sequtils,httpclient],
+  std/[tables,strutils,times,sequtils],
   chronicles,
   chronicles/topics_registry,
   chronos,
+  chronos/timer as ctime,
```

Comment:
For interest's sake, I remember that I also earlier needed both `chronos/timer` and `std/times` for some use cases before (though I can't remember details). Generally we avoid `std/times`, but there are still places where it may be necessary. I'm hoping we can abstract all of these use cases to our own `common/time` in future, as there's been some confusing issues that arose from different time impls in the past.

Reply:
Interesting will take it into account :)

In my case I imported `std/times` to use `getTime()`. The chronos alternative would be `Moment.now()` I guess, but since I needed a readable time, getTime() made the work.

```nim
import chronos
import std/times

echo Moment.now() #533979900656875ns
echo getTime() # 2022-11-16T16:01:20+01:00
```